### PR TITLE
Track ga event for refresh button

### PR DESCRIPTION
**Pull request by jishaal**

### CODE REVIEW

File: app/components/LatestDataRefresh/LatestDataRefresh.tsx [hunk 1]
```diff
@@ -41,7 +42,10 @@ export const LatestDataRefresh = ({
     >
       <Button
         className={cx(styles["refresh-button"], "animate-fade-in")}
-        onClick={() => router.reload()}
+        onClick={() => {
+          trackEvent({ action: "refresh-latest-data" });
```
Comment (jishaal) [inline app/components/LatestDataRefresh/LatestDataRefresh.tsx:46]: Should it be `refresh_latest_data` to follow GA's convention? 🤷🏼‍♂️

**Review by finnclark — APPROVED** [review]

File: app/utilities/trackEvent.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+type GTagEvent = {
+  action: string;
+  category?: string;
+  label?: string;
+  value?: number;
+};
+
+const trackEvent = ({ action, category, label, value }: GTagEvent): void => {
+  window.gtag("event", action, {
```
Comment (andycarrell) [inline app/utilities/trackEvent.ts:9]: _Probably_ want to do a `window` undefined check here just in case ;)

Comment (finnclark) [reply app/utilities/trackEvent.ts:9]: Whoopsie sorry I just merged it last night. Want to create a new PR and we can chuck it in before the button goes up later today?

Comment (andycarrell) [reply app/utilities/trackEvent.ts:9]: That's all good! It's probably being extra defensive 💃 but yeh will do 1 sec

**Pull request merged**